projects/sensory_interface/public/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Why do you need `index.html` twice?
If you need it in `public`, you can remove it from the parent folder. Same for `app.js`.